app/src/main/java/org/fossasia/susi/ai/signup/SignUpActivity.kt

```diff
@@ -214,9 +214,9 @@ class SignUpActivity : AppCompatActivity(), ISignUpView {
 
     private fun signUpToTermsConditionPage() {
         signUpToTermsCondition.setOnClickListener {
-            val intent = Intent(this@SignUpActivity, SkillsActivity::class.java)
-            intent.putExtra(Constant.SIGN_UP_TO_PRIVACY, true)
-            startActivity(intent)
+            val intentToTermsCondition = Intent(this@SignUpActivity, SkillsActivity::class.java)
```

Comment:
Needless change

Reply:
Sir, I didn't understand what changes you wanted in the last comment. Can you please clarify?

Reply:
Revert this to intent